exercises/practice/after-party/after_party_test.py

```diff
@@ -0,0 +1,95 @@
+# These tests are auto-generated with test data from:
+# https://github.com/exercism/website/blob/main/bootcamp_content/projects/string-puzzles/exercises/after-party/config.json
+# File last updated on 2024-02-28
+
+import unittest
+from after_party import (
+    on_guest_list,
+)
+
+
+class AfterPartyTest(unittest.TestCase):
+    def test_empty_list(self):
+        """
+        slug: empty-list
+        description: No-one's allowed in
+        name: Empty register
+        """
+        candidates, name_to_check = [], "Brad"
+        expected = False
```

Comment:
Is there value to having an `excepted` variable? You might be able to directly do an `assertTrue()` or `assertFalse()` for this exercise.